pwiz_tools/Skyline/FileUI/PeptideSearch/ImportPeptideSearchDlg.cs

```diff
@@ -567,22 +567,32 @@ private void NextPage()
                     {
                         if (!BuildPepSearchLibControl.PerformDDASearch)
                         {
-                             HasPeakBoundaries = BuildPepSearchLibControl.SearchFilenames.All(f => f.EndsWith(BiblioSpecLiteBuilder.EXT_TSV));
-                                if (BuildPepSearchLibControl.SearchFilenames.Any(f => f.EndsWith(BiblioSpecLiteBuilder.EXT_TSV)) && !HasPeakBoundaries)
-                                {
-                                    MessageDlg.Show(this, PeptideSearchResources.ImportPeptideSearchDlg_NextPage_Cannot_build_library_from_OpenSWATH_results_mixed_with_results_from_other_tools_);
-                                    return;
-                                }
+                            HasPeakBoundaries = BuildPepSearchLibControl.SearchFilenames.All(f =>
+                                f.EndsWith(BiblioSpecLiteBuilder.EXT_TSV));
+                            if (BuildPepSearchLibControl.SearchFilenames.Any(f =>
+                                    f.EndsWith(BiblioSpecLiteBuilder.EXT_TSV)) && !HasPeakBoundaries)
+                            {
+                                MessageDlg.Show(this,
+                                    PeptideSearchResources
+                                        .ImportPeptideSearchDlg_NextPage_Cannot_build_library_from_OpenSWATH_results_mixed_with_results_from_other_tools_);
+                                return;
                             }
                         }
+                    }
 
                     var eCancel = new CancelEventArgs();
                     if (!BuildPepSearchLibControl.PerformDDASearch && !BuildPeptideSearchLibrary(eCancel, IsFeatureDetectionWorkflow))
                     {
-                        // Page shows error
                         if (eCancel.Cancel)
+                        {
+                            // Page has shown an error and canceled further progress
                             return;
+                        }
+                        // A failure has occurred
+                        // CONSIDER: Should this ever cause the wizard to be closed and force the user to start over?

```

Comment:
I don't understand the "Should this ever cause the wizard to be closed" part of this comment. It looks like it always causes the wizard to be closed. Is there a typo?

Reply:
Think of CONSIDER as not quite as forceful as a TODO. This code maybe should be changed, but I am leaving it as it was when I found it. I am not sure why we want to close the wizard. That may always be something a user would prefer we not do. But this code existed this way before. I am not changing the CloseWizard part. Just saying it should be reconsidered in the future.